exporter/metric/metric.go

```diff
@@ -418,12 +418,10 @@ func recordToMdpbKindType(r *export.Record) (googlemetricpb.MetricDescriptor_Met
 
 	var kind googlemetricpb.MetricDescriptor_MetricKind
 	switch mkind {
-	case apimetric.CounterKind, apimetric.UpDownCounterKind:
+	case apimetric.CounterKind, apimetric.UpDownCounterKind, apimetric.SumObserverKind, apimetric.UpDownSumObserverKind:
```

Comment:
@james-bebbington could you please check if this mapping is correct?

Reply:
We do still need to figure out how to handle `UpDownCounters` and `UpDownObservers` as I don't believe these will show up correctly in Cloud Monitoring as `Cumulatives` (which are assumed to always go up).

I think it's okay to merge this for now as `SumObservers` should definitely be treated as `Cunmulatives` and that is the primary change in this PR. Could you please add a TODO to the code to stating that we need to confirm how UpDownCounters should be handled.